utils/logalloc.cc

```diff
@@ -1987,11 +1987,15 @@ region& region::operator=(region&& other) {
 region::~region() {
 }
 
-occupancy_stats region::occupancy() const {
+occupancy_stats region::occupancy() const noexcept {
     return get_impl().occupancy();
 }
 
-region_group* region::group() {
+const region_group* region::group() const noexcept {
+    return get_impl().group();
+}
+
```

Comment:
This is more than marking something noexcept.

Reply:
Right, sorry.  I'll separate it into a patch of its own.

Reply:
split out in v9